.github/dependabot.yml

```diff
@@ -5,7 +5,7 @@ updates:
     schedule:
       interval: "daily"
     assignees:
-      - "Quantalabs"
+      - "epispot"
```

Comment:
@epispot is not assignable; mention a team instead, like @epispot/maint

Reply:
Fixed